lib/orchestrator.js

```diff
@@ -54,6 +54,30 @@ var Orchestrator = function (url, token, tempLoc, httpLoc, resultLoc) {
 };
 
 /**
+ * Pseudo-constant to indicate a step that finished OK.
+ */
+
+Orchestrator.STATUS_OK = 'ok';
+
+/**
+ * Pseudo-constant to indicate successful completion.
+ */
+
+Orchestrator.STATUS_SUCCESS = 'success';
+
+/**
+ * Pseudo-constant to indicate failure.
+ */
+
+Orchestrator.STATUS_FAILURE = 'failure';
+
+/**
+ * Pseudo-constant to indicate errors.
+ */
+
+Orchestrator.STATUS_ERROR = 'error';
+
```

Comment:
Aaaaall this noise around the actual code :-)

```js
Orchestrator.STATUS_OK = 'ok';
Orchestrator.STATUS_SUCCESS = 'success';
Orchestrator.STATUS_FAILURE = 'failure';
Orchestrator.STATUS_ERROR = 'error';
```

is way enough considering the comments are just copy and paste of the keywords themselves :-)